Operational-Procedures/Scheduler/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _scheduler-operational-procedures:
+
+################################
+Scheduler Operational Procedures
+################################
+
+The Scheduler CSC is in charge of computing an observing queue for the observatory.
+There are two instances of the CSC running at all times, one dedicated to the Main Telescope and another to the Auxiliary Telescope, with indexes 1 and 2 respectively.
+
+Both CSCs share the exact same code-base, which means operating and throubleshooting shares several commonalities.
```

Comment:
a few of grammar/typos.
...which means both operating and troubleshooting share several commonalities.